src/app/auth/reset-password/reset-password.page.ts

```
@@ -53,7 +53,7 @@
 
     this.routerAuthService
       .sendResetPassword(email)
-      .pipe(tap(() => (this.isLoading = false)))
+      .pipe(finalize(() => (this.isLoading = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)